samples/sid_end_device/src/sidewalk_events.c

```diff
@@ -326,6 +353,7 @@ void sidewalk_event_exit(sidewalk_ctx_t *sid, void *ctx)
 		list_element = sys_slist_get(&pending_message_list);
 	}
 	k_mutex_unlock(&pending_message_list_mutex);
+	sid->handle = NULL;
```

Comment:
Isn't it handled by sid api (init/deinit)?

Reply:
sid api can not NULL this pointer on its own.